types/office-js-preview/index.d.ts

```diff
@@ -1006,6 +1006,20 @@ declare namespace Office {
          * Provides objects and methods that you can use to create and manipulate UI components, such as dialog boxes.
          */
         ui: UI;
+        /**
+         * Gets the object to retrieve the runtime URL of an add-in.
```

Comment:
The description reads as if it's for a singular URL, but the property is plural. Seems inconsistent.

Reply:
Fixed.